ajax/libs/jquery.tablesorter/package.json

```diff
@@ -17,10 +17,7 @@
     "alphanumeric",
     "natural"
   ],
-  "licenses": [
-    "MIT",
-    "deprecated_GPL-2.0+"
-  ],
+  "licenses": "(MIT OR GPL-2.0+)",
```

Comment:
Oh, I thought you wanted a SPDX string... I should have used "license" (no "s").